File: example/javalib/testing/5-test-parallel/foo/test/src/foo/RandomTestsUtils.java
package foo;

import static org.junit.Assert.assertEquals;

public class RandomTestsUtils {
  protected void testGreeting(String name, int sleepTime) throws Exception {
    String greeted = Foo.greet(name);
    Thread.sleep(sleepTime);
    assertEquals("Hello " + name, greeted);
  }
}

File: example/kotlinlib/testing/6-test-group-stealing/foo/test/src/foo/RandomTestsUtils.kt
package foo

import org.junit.jupiter.api.Assertions.assertEquals

open class RandomTestsUtils {
    @Throws(Exception::class)
    protected fun testGreeting(name: String, sleepTime: Int) {
        val greeted = Foo.greet(name)
        Thread.sleep(sleepTime.toLong())
        assertEquals("Hello $name", greeted)
    }
}
